data/tokens.json

```diff
@@ -1930,6 +1930,20 @@
       "type": "typography"
     }
   },
+  "date picker": {
+    "gtmds_date-picker_margin": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_date-picker_padding-horizontal": {
+      "value": "{gtmds_space-sm}/2",
+      "type": "spacing"
+    },
+    "gtmds_date-picker_padding-vertical": {
+      "value": "{gtmds_space-xxs}/2",
```

Comment:
```suggestion
      "value": "{gtmds_size-sm}",
```

Reply:
Still in the works, but I will update this. Good reminder.